daemons/gptp/gptp_cfg.ini

```diff
@@ -18,19 +20,23 @@ syncReceiptTimeout = 3
 # various links, in clause 11.2.2. It needs to be configurable because a
 # user may be using a converter to fibre and the software wont know until the
 # administrator changes it.
-neighborPropDelayThresh = 800
+neighborPropDelayThresh = 8000
```

Comment:
I did that on october, and I think I was having trouble to synchronize it against one Extreme switch. I will keep the default to 800 though.